managed/src/main/java/com/yugabyte/yw/models/Universe.java

```diff
@@ -275,8 +275,11 @@ public static boolean checkIfUniverseExists(String universeName) {
    * WARNING: Returns partially filled Universe objects!!
    * @return list of UUIDs of all universes
    */
-  public static List<Universe> getAllUuids() {
-    return find.query().select("universeUUID").findList();
+  public static Set<UUID> getAllUuids() {
```

Comment:
1> This function was buggy - it did not support multi-tenant use case. You should look for UUIDs only in current customer.
2> selecting universeUUID and getting list of universes then processing it to get UUID list is very roundabout. Use findIds instead.

I believe I have fixed this at head during another change I was working on. So just sync.

Reply:
Sure.

Reply:
I do not think I can reuse the same in this PR as we need to get the expired backups independent of the customer.